node/docs/nodeVersioning.md

```diff
@@ -0,0 +1,20 @@
+# Node versioning
+
+## Agent Node Handler
+
+The agent currently has 2 different node handlers that it can use to execute node tasks: Node 6 and Node 10.
+The handler used depends on the `execution` property specified in the tasks `task.json`.
+If the `execution` property is specified to be `Node`, the task will run on the Node 6 handler, if it is specified to be `Node10` it will run on the Node 10 handler.
+Some legacy agents will use a Node 5 handler if the `execution` is specified to be `Node`, but this is a very small percentage of existing agents.
```

Comment:
let's just take this out.  It's so long ago it's probably more of a distraction.  The only agents that are doing this are likely on prem not even dealing with these new tasks.